src/translations/sqlb_nl.ts

```diff
@@ -0,0 +1,7344 @@
+        <translation>Tijdelijke</translation>
+    </message>
+    <message>
+        <location filename="../DbStructureModel.cpp" line="327"/>
+        <source>Tables (%1)</source>
+        <translation>Tabellen (%1)</translation>
+    </message>
+    <message>
+        <location filename="../DbStructureModel.cpp" line="332"/>
+        <source>Indices (%1)</source>
+        <translation>Indices (%1)</translation>
+    </message>
+    <message>
+        <location filename="../DbStructureModel.cpp" line="337"/>
+        <source>Views (%1)</source>
+        <translation>Views (%1)</translation>
+    </message>
+    <message>
+        <location filename="../DbStructureModel.cpp" line="342"/>
+        <source>Triggers (%1)</source>
+        <translation>Triggers (%1)</translation>
+    </message>
+</context>
+<context>
+    <name>EditDialog</name>
+    <message>
+        <location filename="../EditDialog.ui" line="14"/>
+        <source>Edit database cell</source>
+        <translation>Databasecel bewerken</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="17"/>
+        <source>This area displays information about the data present in this database cell</source>
+        <translation>Dit gebied toont informatie over de aanwezige gegevens in de databasecel</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="43"/>
+        <source>Mode:</source>
+        <translation>Modus:</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="59"/>
+        <source>This is the list of supported modes for the cell editor. Choose a mode for viewing or editing the data of the current cell.</source>
+        <translation>Dit is de lijst van ondersteunde modi voor de celbewerker. Kies een modus om de gegevens van de huidige cel te bekijken of te bewerken.</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="63"/>
+        <source>Text</source>
+        <translation>Tekst</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="68"/>
+        <source>RTL Text</source>
+        <translation>Rechts-naar-linkstekst</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="73"/>
+        <source>Binary</source>
+        <translation>Binair</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="78"/>
+        <location filename="../EditDialog.cpp" line="255"/>
+        <source>Image</source>
+        <translation>Afbeelding</translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="83"/>
+        <source>JSON</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../EditDialog.ui" line="88"/>
+        <source>XML</source>
+        <translation></translation>
```

Comment:
I think it's a proper noun that doesn't need translation, so I think you've left the translation blank.
I also did like this in the past, but there was an comments that it is safer to copy the original text for proper nouns
that do not require translation. Please refer to [here](https://github.com/sqlitebrowser/sqlitebrowser/pull/2329#issuecomment-658202890) for details.